source/translators/concepts/translation-process.rst

```diff
@@ -0,0 +1,84 @@
+Translations Process
+####################
+
+High Level
+**********
+
+Your code needs to annotate strings so tooling can know that you want those strings
+translated. The tooling needs to extract those strings in order to be shared with
+translators. Once the strings have been extracted and made available to translators,
+strings can be translated to a variety of languages. Tooling will then download the
+translated strings and make them available to your running code. 
+
+.. mermaid::
+
+   flowchart TD
+      annotate(Annotate Strings) -->
+      extract(Extract Strings For Translation) -->
+      upload(Upload to Transifex) -->
+      translate(Translate Strings) -->
+      download(Download for Use)
+
+Annotating and Extracting Strings
+*********************************
+
+Given code like this:
+
+.. code-block::
+
+   from django.utils.translation import ugettext as _
+
+   # Translators: This will help the translator
+   message = _("Welcome!")
+
+   # Translators: This is a very long comment that needs to wrap
+   # over multiple lines because it would be too long otherwise.
+   message = _("Hello world")
+
+Our tooling will extract the following strings:
+
+* ``Welcome!``
+
+* ``Hello world``
+
+For our django backed services, we use `django's builtin translation tooling`_ to extract
+strings for translation. In our react based frontends, we use react-intl_ to extract
+strings for translation.
+
+To learn more about how to annotate strings for translation, see the annotation reference
+(link tbd).
```

Comment:
We wrote https://github.com/openedx/docs.openedx.org/issues/211 to track this work, maybe put a link to that in a comment here.

Reply:
Added a comment with the link to the issue.